generator/generic/templates/sequence_definition.rb.erb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Inferno
+  module USCore310ProfileDefinitions
```

Comment:
@czh-orz renamed this to `ProfileDefinitions`

Reply:
It can't just be `ProfileDefinitions` because then every generated sequence will share the same definitions. It needs to be `Inferno::SOMETHING_UNIQUEProfileDefinitions` or `Inferno::SOMETHING_UNIQUE::ProfileDefinitions`.

Reply:
Renamed so it includes the `path`.  If the module is in the `ig` directory it will be named `IgProfileDefinitions` and if its in the `uscore` directory it will be `UscoreProfileDefinition`.